test/dynamo/cpython/3_13/typinganndata/ann_module.py

```diff
@@ -50,7 +49,7 @@ class S(metaclass = Meta):
 
 def foo(x: int = 10):
     def bar(y: List[str]):
-        x: str = 'yes'
+        pass
```

Comment:
Isn't the intent of the test to check this?

Reply:
yes, this is an incident change that must be reverted..